### PR TITLE
fixes

### PR DESCRIPTION
**Pull request by HadiEsmaeili86**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# .github
```
Comment (Sojaner) [inline README.md:1]: Add a line underneath the line above and add `---` to it

**Review by Sojaner — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+# .github
```
Comment (HadiEsmaeili86) [inline README.md:1]: New add

**Review by Sojaner — CHANGES_REQUESTED** [review]
Just one line of `---` is enough. :)

**Review by Sojaner — APPROVED** [review]
Looks correct now

**Pull request merged**